samples/lj_liquid.py

```diff
@@ -133,22 +136,16 @@
 
 # Warmup Integration Loop
 i = 0
-while (i < warm_n_times and act_min_dist < min_dist):
+while i < warm_n_times and act_min_dist < min_dist:
```

Comment:
maybe we should ```minimize_energy``` here

Reply:
If we should, we should system.integrator.set_steepest_descent()

Reply:
Done in 23d39c7 for all samples.